### ISSUE TITLE
Issue with some games

### ISSUE THREAD

**Issue by monadosorin** [bucanero/apollo-ps4#33]
Hello i've been using this tool for a while now and it works great most of the time. but i do run into a few issue here and there. for example tales of arise works great at first but i can't save, i can't load after the first time, and for some reason changing the voice language crash it? another example is horizon zero dawn complete edition. it assigns fine on the save tool but the saves doesn't show up on the game for some reason. for horizon zero dawn there's multiple save files, there's probably 7 or more i think. most of them are called a mission name for example "[12] the heart of the nora" but one of the save file is just named "horizon zero dawn complete edition". i tried re assigning the saves in different order and such but that doesn't work either so yeah, i hope there's solution to this because i accidentally deleted my save file and i want to restore it somehow :')

**Comment by Masamune3210** [comment]
save name is dictated by the param.sfo, does this rewrite them? not sure how else the save name would change unless the system is just incorrectly reading them

**Comment by monadosorin** [comment]
Does the name effect if the saves work or not? And I don't think the save names changes either

**Comment by Masamune3210** [comment]
oh i misread the issue, the one save named like the game is probably normal. most games have at least one generic save to handle things like options and overall progression stats. it shouldn't affect the game being able to load the save as that is handled by the titleid and the keystone

**Comment by monadosorin** [comment]
Oh okay, so do i still need to assign the save file? I wonder why horizon zero dawn doesn't work though, huh

**Comment by bucanero** [comment]
if you're mixing retail and fpkg saves, be sure to check that the keystones match the right game, or else the saves won't work properly.

**Comment by monadosorin** [comment]
it's actually my own save file i backed up. i accidentally transfered my older usb save to my hdd and it overwritten it. i tried re assinging it but still no luck

**Comment by MindlessToxin423** [comment]
This sounds similar to the issue I'm having with HZD Complete Edition. I did look at the save details and the one that's titled "Horizon Zero Dawn Complete Edition" it's a profile save. There's actually 11 saves on mine for CUSA01021. The one profile save, nine quicksaves, and one manual save. I've tried moving just the manual and latest quicksave, manual with all quicksaves, also tried with and without the profile save.
